src/main/java/com/specialwarriors/conal/user/domain/User.java

```diff
@@ -24,6 +25,21 @@ public class User {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(nullable = false)
+    private Long githubId;
```

Comment:
```suggestion
    private long githubId;
```
기본 타입으로 정의하면 `not null` 조건이 자동 적용되는 것으로 알고 있는데 래퍼 타입으로 선언하고 어노테이션으로 추가 설정하신 이유가 궁금합니다.

Reply:
컬럼을 래퍼타입으로 선언하는게 습관이라 githubId도 Long으로 선언했습니다. long으로 수정하는게 좋을까요?